docs/02-membership.md

```diff
@@ -276,6 +249,78 @@ The table and graphs below illustrate how the 5 year weight change of a hypothet
 
 ![RelativeWeights](https://github.com/cheeky-gorilla/membership/assets/76262359/93a6f5a0-393e-46c8-8dfd-a51aa6864f46)
 
+### Governance
+
+Only members are invited to govern the mechanism. This may include:
```

Comment:
The phrasing here feels too "outside looking", I'd reframe as "Protocol Guild members govern the evolution of the mechanism, which includes:"

I'd also add a note about how governance does **not** have control over any vesting funds. PG members can't vote to "cancel the vest and get all the tokens now".